outer.c

```diff
@@ -207,6 +263,21 @@ static void persist_ns_files(pid_t pid, const char **persist)
    host system, so we reimplement that functionality here. */
 void outer_helper_spawn(struct outer_helper *helper)
 {
+	pid_t rootpid = getpid();
+
+	// If cleanup is needed, fork process to epoll cgroup.events
+	if (helper->cleanfd != -1) {
+		pid_t pid = fork();
+		if (pid == -1) {
+			err(1, "outer_helper: cgroup cleanup fork");
+		}
+
+		if (pid == 0) {
+			cgroup_helper(helper->cleanfd, rootpid);
+			_exit(0);
+		}
```

Comment:
I'd maybe leave a comment saying that we explicitly let the process leak in order for the cleanup to happen after bst exits.

Reply:
Done